pytransform3d/transform_manager/_temporal_transform_manager.py

```diff
@@ -44,14 +48,84 @@ class StaticTransform(TimeVaryingTransform):
     def __init__(self, A2B):
         self._A2B = A2B
 
-    def as_matrix(self, time):
+    def as_matrix(self, query_time):
         return self._A2B
 
     def check_transforms(self):
         self._A2B = check_transform(self._A2B)
         return self
 
 
+class NumpyTimeseriesTransform(TimeVaryingTransform):
+    """Transformation sequence, represented in a numpy array.
+    The interpolation is computed using screw linear interpolation (ScLERP)
+    method.
+
+    Parameters
+    ----------
+    time_arr: np.ndarray, shape (N,)
+        Numeric timesteps corresponding to the transformation samples.
+        You can use unixtime, relative time (starting with 0.0).
+    pq_arr : pandas.DataFrame, shape (N, 7)
+        Time-sequence of transformations, with each row representing a single
+        sample as position-quarternion (PQ) structure.
+    """
+
+    def __init__(self, time, pqs):
+
+        if len(pqs.shape) != 2:
+            raise ValueError("Shape of PQ array must be 2-dimensional.")
```

Comment:
Are there test cases for these ValueErrors?

Reply:
Added